api-platform/core/3.0/config/packages/api_platform.yaml

```diff
@@ -0,0 +1,11 @@
+api_platform:
+    title: Hello API Platform
+    version: 1.0.0
+    # Mercure integration, remove if unwanted
+    mercure: ~
+    # Good cache defaults for REST APIs
```

Comment:
If these are good defaults, why aren't they the default value in ApiPlatformBundle directly ?

Reply:
Actually, these are good defaults for any Symfony application... This should be in FrameworkBundle. I don't want API Platform to override Symfony's default silently, it's why it's in the recipe and not in the bundle.

Reply:
But those are an ApiPlatform configuration setting, not a FrameworkBundle one.